enlace/include/dosage_percentage.php

```diff
@@ -94,14 +99,27 @@ function calculate_dosage ( $session, $participant, $start_date, $end_date, $exc
         INNER JOIN Participants ON Participants_Programs.Participant_ID=Participants.Participant_ID
         LEFT JOIN Absences ON ( Program_Date_ID = Program_Date AND Participants_Programs.Participant_ID=
```

Comment:
I think this was a `LEFT JOIN` before because there wouldn't be an entry in `Absences` for the days that a participant attended a session.  Now that there is an entry either way, should this be an `INNER JOIN`?

Reply:
Good catch!  I also changed the previous query to match.